com.unity.render-pipelines.high-definition/Documentation~/Mask-Map-and-Detail-Map.md

```diff
@@ -32,7 +32,9 @@ The following example image demonstrates the individual components of a full mas
 
 ## Detail map
 
-The detail map contains two grayscale Textures and one two-component Texture, which is the Material's normal map.
+The detail map allow you to overlay a second set of textures on top of the base surface information. Typically, the detail map would be scaled several times across the object’s surface to add small details to a material.
+The detail map contains two grayscale Textures and one two-component Texture, which is the Material's detail normal map.
+When importing the texture, make sure the `sRGB` checkbox is not ticked in the Import Settings.
```

Comment:
When you import the detail map texture, make sure to disable the **sRGB** checkbox in the Import Settings.